commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java

```diff
@@ -82,10 +82,10 @@ public void afterEach() {
     Screenshots.finishContext();
   }
 
-  @AfterEach
+ /* @AfterEach
   public void close() {
     if (!title().equals("Login")) {
       Selenide.close();
     }
-  }
+  }*/
```

Comment:
Why this change ? Shouldn't we keep this to make sure the user is loggued in ? (cc @ilyeskallala )

Reply:
This change to make sure that the user is loggued out. if it's not the case, the browser is closed and reopened for the next TC in login page.

Reply:
So we keep this change ? If so, no need to keep it in comments, we can delete it.